js/npm/parity/package.json

```diff
@@ -26,8 +26,8 @@
   "devDependencies": {
   },
   "dependencies": {
-    "bignumber.js": "^2.3.0",
-    "js-sha3": "^0.5.2",
-    "node-fetch": "^1.6.3"
+    "bignumber.js": "~2.3.0",
```

Comment:
Why not using `2.3.0` directly? I'm sure the patches won't break anything, but we never know...